src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.jsx

```diff
@@ -67,6 +70,7 @@ const RadioLineButtonFill = styled.div`
   border-radius: 50%;
   background-color: ${p => p.theme.green};
   animation: ${p => (p.animate ? `0.2s ${growIn} ease` : 'none')};
+  opacity: ${p => (p.isDisabled ? 0.4 : null)};
```

Comment:
This is similar to what we do for switches.

Reply:
Should prop name just be `disabled`? idk which is more consistently used.

Reply:
If I name it `disabled` emotion will forward it to the DOM since it's a valid html attribute. But emotion isn't smart enough to know that `div`'s shouldn't have the `disabled` attribute.